### DISCUSSION TITLE
About Rest API and webservice  agaom

### DISCUSSION THREAD

**Discussion by gsportelli73** [YetiForceCompany/YetiForceCRM#15158] [General]
Hello, i have created a rest api to insert products in CRM, now in product i have a category tree.
In this field i have seem that to choice i category from rest Api in my Json file i must insert key T1, T2, ecc. 
This is ok. 
But i want to create i category from rest api is it possibile ?
I have wtached in API but i have understand how to do this. 
Thank you.

**Comment by rskrzypczak** [comment]
This is not possible at the moment.

**Comment by mariuszkrzaczkowski** [comment]
Please confirm if I understand your question,
Do you want to add a new value to the category field?

**Comment by gsportelli73** [comment]
Yes, i do

**Reply by gsportelli73** [reply]
Hello, yes i do

**Reply by mariuszkrzaczkowski** [reply]
This is not possible at the moment.

**Comment by gsportelli73** [comment]
Thank you
It is a I portant feature to solve synchronization between two application
crm and other software.
We are writing a gateway software that send data between software and crm
and vice versa. It is ok to import client customer product and some custom
field in crm.
My idea is create a universal gateway to link crm and any software as
account and warehouse types.


Il gio 9 set 2021, 13:00 Mariusz Krzaczkowski ***@***.***> ha
scritto:

**Reply by mariuszkrzaczkowski** [reply]
The API for managing the CRM system requires significant changes in building the administration panel, we currently do not have it in the nearest plans

**Comment by gsportelli73** [comment]
Ok I think new solution if I use module product category?
I have tried to create a field  whitin product module.
It is a module field, when I querying via rest api field list it is not
present. Is it correct?


Il gio 9 set 2021, 23:18 Mariusz Krzaczkowski ***@***.***> ha
scritto:

**Reply by mariuszkrzaczkowski** [reply]
using the new module you will be able to manage via API

**Comment by gsportelli73** [comment]
Hello I create a new module within product? Is it correct?
Or do you think a new module outside enturt product for example?

Il ven 10 set 2021, 08:03 Mariusz Krzaczkowski ***@***.***>
ha scritto:

**Reply by mariuszkrzaczkowski** [reply]
use the product categories module

**Comment by gsportelli73** [comment]
Hello I have understand but how to see this module in entity product via
rest api?

Il ven 10 set 2021, 21:07 Mariusz Krzaczkowski ***@***.***>
ha scritto:
